### ISSUE TITLE
Rebar generate not work if erlang compile manualy (Ubuntu)

### ISSUE THREAD

**Issue by Slavenin** [rebar/rebar#415]
I`am clone erlang from github and compile it. In my erlang folder in lib directory no version erlang library. Only simpe name (asn, common_test, ... reltool (not reltool-0.6.6) etc).  If i run 
```
rebar compile generate
```
i see error:
```
ERROR: Reltool support requires at least reltool-0.5.2; this VM is using reltool
```

Sorry for my english.

**Comment by tuncer** [comment]
@Slavenin does #418 work for you?

**Comment by tuncer** [comment]
Fixed in #418.
